libs/domain/user/account-menu/account-menu.component.ts

```diff
@@ -0,0 +1,43 @@
+import { ContextController } from '@spryker-oryx/core';
+import { ButtonType } from '@spryker-oryx/ui/button';
+import { IconTypes } from '@spryker-oryx/ui/icon';
+import { AccountMenuLinks } from '@spryker-oryx/user';
+import {
+  I18nMixin,
+  hydrate,
+  signal,
+  signalAware,
+} from '@spryker-oryx/utilities';
+import { LitElement, TemplateResult, html } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { styles } from './account-menu.styles';
+
+@signalAware()
+@hydrate({ event: 'window:load' })
+export class AccountMenuComponent extends I18nMixin(LitElement) {
+  static styles = styles;
+  protected links = [
+    { id: AccountMenuLinks.OVERVIEW, icon: IconTypes.Description },
```

Comment:
Sorry, I'm out of the context. Is this hardcoded list a temporary solution? How to extend or customise it?

Reply:
yeah we should get this list from props or options

Reply:
It's dynamic now.